engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -3149,8 +3155,14 @@ namespace dmGameSystem
         gui_context->m_MaxAnimationCount = dmConfigFile::GetInt(ctx->m_Config, "gui.max_animation_count", 1024);
         gui_context->m_MaxParticleBufferCount = dmConfigFile::GetInt(ctx->m_Config, "gui.max_particle_buffer_count", 1024);
 
-        int32_t max_gui_count = dmConfigFile::GetInt(ctx->m_Config, "gui.max_instance_count", 128);
-        gui_context->m_Worlds.SetCapacity(max_gui_count);
+        // TODO: it seems like this is a hidden game.project property which should be gui.world_count like it is in physics
+        int32_t max_world_count = dmConfigFile::GetInt(ctx->m_Config, "gui.max_instance_count", 128);
+        if (max_world_count > 4096)
```

Comment:
it seems like this is a hidden option. I added a comment here, not sure if I should fix it (make it visible in the game.project + rename it) as part of this PR

Reply:
I'm not sure why it's hidden. Was it ever present and then removed? (i.e. deprecated?)